lib/src/components/NxList/NxListItem.tsx

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+import React, { forwardRef } from 'react';
+import classnames from 'classnames';
+import {NxListItemProps, nxListItemPropTypes} from './types';
+
+const NxListItem = forwardRef<HTMLLIElement, NxListItemProps>(
+    function NxListItem({ children, className, ...attrs }, ref) {
+      const classNames = classnames('nx-list__item', className),
+          listItem = (
+            <li ref={ref}
+                className={classNames}
+                {...attrs}>
+              {children}
+            </li>
+          );
+      return listItem;
```

Comment:
Any particular reason the JSX is defined in a separate statement from the `return`?

Reply:
> Any particular reason the JSX is defined in a separate statement from the `return`?

That was an error on my part and has been fixed.
bba7c8ced960e79f995cc7544f363087cebcca9e